pkg/api/v1beta1/types.go

```diff
@@ -710,17 +713,18 @@ type Namespace struct {
 	Labels map[string]string `json:"labels,omitempty" description:"map of string keys and values that can be used to organize and categorize namespaces"`
 
 	// Spec defines the behavior of the Namespace.
-	Spec NamespaceSpec `json:"spec,omitempty"`
+	Spec NamespaceSpec `json:"spec,omitempty" description:"spec defines the behavior of the Namespace"`
```

Comment:
I don't have a ton more time to refine this today, so if the goal is to enable the script to not allow more deviation, then this should be good to go.  We can nit text later I guess.